katello/katello/db_config.rb

```diff
@@ -0,0 +1,66 @@
+require 'yaml'
+
+module KatelloUtilities
+  class DBConfig
+    attr_reader :foreman, :candlepin
+    INSTALLER_CONFIG = '/etc/foreman-installer/scenarios.d/last_scenario.yaml'
+
+    def initialize(scenario = INSTALLER_CONFIG)
```

Comment:
This function could use some error handling. What if the scenario file doesn't exist? What if it's on a proxy where foreman or katello is disabled?